contrib/generate_precompile.jl

```diff
@@ -127,8 +131,7 @@ precompile_script = """
 
 julia_exepath() = joinpath(Sys.BINDIR, Base.julia_exename())
 
-have_repl =  haskey(Base.loaded_modules,
```

Comment:
@KristofferC does PackageCompiler use `generate_precompile.jl`?
I haven't deleted the code here since someone may still want to create a fully baked sysimg?

We can probably put the ` haskey(Base.loaded_modules,...` stuff back in to continue to support such users.

Reply:
PackageCompiler does not use it.

Reply:
Yeah I saw that in https://github.com/JuliaLang/julia/pull/51189 you just removed them.